src/Paramore.Brighter.MessagingGateway.AWSSQS/ValidateTopicByArnConvention.cs

```diff
@@ -35,11 +35,22 @@ public class ValidateTopicByArnConvention : ValidateTopicByArn, IValidateTopic
         private readonly RegionEndpoint _region;
         private AmazonSecurityTokenServiceClient _stsClient;
 
-        public ValidateTopicByArnConvention(AWSCredentials credentials, RegionEndpoint region) : base(credentials, region)
+        public ValidateTopicByArnConvention(AWSCredentials credentials, RegionEndpoint region, Action<ClientConfig> clientConfigAction = null) 
+            : base(credentials, region, clientConfigAction)
         {
             _region = region;
 
-            _stsClient = new AmazonSecurityTokenServiceClient(credentials, region);
+            var config = new AmazonSecurityTokenServiceConfig
```

Comment:
Again should we replicate the pattern of making this a method; it's possible if we do that we may be able to extract a helper class that encapsulates this logic. We can just make that a dependency we new up in the constructor, no need to DI it in, but we save on repeated logic amongst these classes.